pkgs/tools/networking/slirp4netns/default.nix

```diff
@@ -1,27 +1,27 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, glib }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, glib, libcap, libseccomp }:
 
 stdenv.mkDerivation rec {
-  pname = "slirp4netns";
-  version = "0.3.0";
+  name = "slirp4netns-${version}";
```

Comment:
Any reason to use `name` in place of `pname`?

(`name` is defined by `mkDerivation` as `${pname}-${version}`)

Reply:
A alright I will change it back.